pkg/distro/defs/fedora/distro.yaml

```diff
@@ -1572,6 +1572,13 @@ image_types:
     iso_label: "Unknown"
     # We don't know the variant that goes into the OS pipeline that gets installed
     variant: "Unknown"
+    # NOTE: this is not supported right now because the
+    # image-installer on Fedora isn't working when unattended.
+    # These options are probably necessary but could change.
+    # Unattended/non-interactive installations are better set to text
+    # time since they might be running headless and a UI is
```

Comment:
I have no idea what "text time" is.  I don't know what I meant when [I wrote the original comment](https://github.com/osbuild/images/commit/e50f259522b020a00e64ece1e1b32a6875ca38c5).  I think I meant to just say "... set to text, since they might ...".

Reply:
I tweaked this now.